azure-pipelines.yml

```diff
@@ -0,0 +1,45 @@
+# Pipeline for the TeamUp's webApp
+
+trigger:
+  # batch runs this pipeline one after the other (useful when there are many contributors)
+  batch: true
+  branches:
+    include:
+      - master
+      - develop
+
+pr:
+  branches:
+    include:
+      - master
+      - develop
+  drafts: false
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '14.17.x' 
+  displayName: 'Installi Node 14.17.x'
+
+# installing dependencies (react, axios, etc)
+- script: npm ci
+  displayName: 'Install dependencies'
+
+# building the webApp
+- script: npm run build
+  displayName: 'Build React webApp'
+#for conditional run of a step or job or stage
+
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Save artifact'
+  inputs:
+    PathtoPublish: '$(Agent.BuildDirectory)/s/build'
+    ArtifactName: 'drop'
+    publishLocation: 'Container'
+#not save artifact on PRs
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
```

Comment:
fix indent

Reply:
on `condition:...` line? it says me that it is and **unexpected property condition** warning